product-service/src/services/getProductById/getProductById.js

```diff
@@ -1,9 +1,16 @@
-import productList from '../../productList.json';
+import { Client } from 'pg';
+import { dbOptions } from '../../utils/dbOptions';
 
 export const getProductById = async (id = '') => {
   if (!id) return null;
-  const foundProduct = productList.find(product => product.id === id);
-  if (!foundProduct) return null;
-
-  return foundProduct;
+  const client = new Client(dbOptions);
+  await client.connect();
+  try {
+    const { rows: foundProduct } = await client.query(`select p.id, p.title, p.description, p.price, p.image, s.count from products p left join stocks s on p.id = s.product_id where p.id = '${id}'`);
+    return foundProduct;
```

Comment:
You return product in array. I suppose it will be much more useful to return an object instead of object in array.